luminance-windowing/src/lib.rs

```diff
@@ -53,14 +53,18 @@ pub enum WindowDim {
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 pub struct WindowOpt {
   hide_cursor: bool,
+  num_samples: Option<u32>,
 }
 
 impl Default for WindowOpt {
   /// Defaults:
   ///
   /// - `hide_cursor(false)`
```

Comment:
We also want to state multisample is disabled by default. I will do it by pushing commits to your branch.